src/components/AppMetaTags/AppMetaTags.js

```diff
@@ -13,32 +13,18 @@ export default function AppMetaTags({
   canonicalUrl,
   pageTitle,
   pageDescription,
-  // BUG: shareTitle and shareDescription don't work! See note below.
-  shareTitle,
-  shareDescription,
```

Comment:
We were using react-helmet (a React component that rewrites your head tags for title, meta tags, etc.) for these sharing tag stuff, but it was ineffective since it only shows up for crawlers that run your JavaScript, which Facebook and Twitter do not.  So I've ripped it out of here and we do it purely in the generate-index-pages script.